src/styles/mimetypes.styl

```diff
@@ -0,0 +1,11 @@
+/*
```

Comment:
@GoOz : I needed classes for all the different file types. I've created this function to generate the css so if we add new file types, we just have to change it here. I hesitated between this and global classes, let me know what you think.